src/DroneSafetyApi/Data/IHazardRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Documents.Spatial;
+using DroneSafetyApi.Models;
+
+
+// Fix namespace in all the data classes and their Dependencies
+namespace DroneSafetyApi.Data
+{
+    public interface IHazardRepository
+    {
+        IEnumerable<Hazard> GetHazardsInRadius<T>(Point point, int radius, string ShapeName, DateTime time) where T : Hazard;
```

Comment:
Can't we have 3 methods in this class? Each of these methods fetches one type of shape from the background (using a string name if we have to). So, for instance,
IEnumerable<Hazard> GetPolygonHazards (...) { ... }. Then this method should call all three of these methods and then concatenate the results together.